workers/python/python/telemetry.py

```diff
@@ -0,0 +1,42 @@
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+from opentelemetry.sdk._logs import LoggerProvider
+
+from opentelemetry import trace
+from opentelemetry._logs import set_logger_provider
+from opentelemetry.exporter.otlp.proto.http._log_exporter import (
+    OTLPLogExporter,
+)
+from opentelemetry.sdk._logs import LoggerProvider, LoggingHandler
+from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    BatchSpanProcessor,
+)
+
+from config import config
+
+resource = Resource(attributes={
+    SERVICE_NAME: "bee-api"
+})
+
+traceProvider = TracerProvider(resource=resource)
+if not config.otel_sdk_disabled:
+    traceProvider.add_span_processor(BatchSpanProcessor(
+        OTLPSpanExporter(endpoint=config.otel_exporter_otlp_endpoint)))
+    trace.set_tracer_provider(traceProvider)
```

Comment:
can we add this code to a `setup_telemetry` function and call it in main? This is configured sort of by accident, just because it's imported in logger.py